Use the correct address for OpenID endpoints for openSUSE

Conan-Kudo opened the pull request:
openSUSE set up Ipsilon in a different configuration, where the
OpenID endpoint isn't `<username>.id.opensuse.org`, but instead a
regular URL path where the username is at the end of the URL.

Conan-Kudo commented:
cc: @hellcp

hellcp commented:
This won't really work anyway, since our openid isn't setup for `$username.id.opensuse.org`, but `www.opensuse.org/openid/user/$username` iirc. I implemented the openid code here before to support such a usecase, but that didn't stay after some changes

Conan-Kudo commented:
@hellcp Any chance we can get this corrected then?

hellcp commented:
Just copy aadd9f03559597b44b572dd71757fbf87bdc51ba

Conan-Kudo commented:
@hellcp How does this look now?

hellcp commented:
Yup, that should work fine

abompard commented on noggin/templates/user-settings-profile.html:
```diff
@@ -12,7 +12,7 @@
     {%- if OPENID is defined %}
     <form method="GET" action="https://www.libravatar.org/openid/login/">
-      <input type="hidden" name="openid_identifier" value="{{ 'https://'+user.username+OPENID }}">
+      <input type="hidden" name="openid_identifier" value="{{ OPENID % format(user.username) }}">
```
This crashes Jinja, it's either `OPENID % user.username` or `OPENID|format(user.username)`. The [docs](https://jinja.palletsprojects.com/en/2.11.x/templates/#format) suggest the former.

Conan-Kudo replied:
This should be fixed now.

Conan-Kudo committed:
Use the correct address for OpenID endpoints for openSUSE

openSUSE set up Ipsilon in a different configuration, where the
OpenID endpoint isn't <username>.id.opensuse.org, but instead a
regular URL path where the username is at the end of the URL.

Signed-off-by: Neal Gompa <ngompa13@gmail.com>
